R/epidist.R

```diff
@@ -1,6 +1,8 @@
 #' Fit epidemiological delay distributions using a `brms` interface
 #'
-#' @param data A `data.frame` containing line list data.
+#' @param data An object with class corresponding to an implemented model.
+#' The currently available models classes are `epidist_latent_model` and
```

Comment:
I'd suggest taking this out and instead linking to somewhere else as it will be easy to get outdated

Reply:
What do you think of this idea https://github.com/epinowcast/epidist/issues/448?

Reply:
[dd6e22c](https://github.com/epinowcast/epidist/pull/447/commits/dd6e22c8548ca8836518e0d645f7b884a9961597)